### PR TITLE
add linked list exercise

### PR DESCRIPTION
**Pull request by kmarker1101**
With this one, I am using "dll" as an abbreviation for doubly linked list. I think this is pretty standard but let me know if it's ok.

### CODE REVIEW

**Review by fapdash — CHANGES_REQUESTED** [review]
Can you please rename `create-dll` to `dll-create` so all the functions follow the same pattern?

**Comment by fapdash** [comment]
> With this one, I am using "dll" as an abbreviation for doubly linked list.

I think it's fine, students will understand from the context.

File: exercises/practice/linked-list/linked-list-test.el
Comment (BNAndras) [inline exercises/practice/linked-list/linked-list-test.el:1]: Can we use`=` instead of `equal` for consistency with other exercises comparing numbers?

**Review by BNAndras — CHANGES_REQUESTED** [review]

File: exercises/practice/linked-list/linked-list-test.el
Comment (kmarker1101) [reply exercises/practice/linked-list/linked-list-test.el:1]: done

**Review by BNAndras — APPROVED** [review]

**Review by fapdash — APPROVED** [review]

**Pull request merged**